.github/workflows/ci_checks.yml

```diff
@@ -127,9 +127,9 @@ jobs:
 
   verify_pr_description:
     runs-on: [ubuntu-latest]
+    if: github.event_name == 'pull_request'
```

Comment:
good catch, I thought I changed this 👍

Reply:
Actually both work, you can skip the job, or a step in the job as well.

Reply:
Yes, but in the previous implementation, I only skipped a step. The empty job still gets triggered, so I wanted to change that.